schema.pgsql

```diff
@@ -16,14 +16,61 @@ CREATE INDEX files_expiry ON files(expiry);
 CREATE TABLE release_versions (
     project varchar(50) PRIMARY KEY,
     version varchar(25) NOT NULL,
+    prerelease_version varchar(25),
     updated TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW()
 );
 
 -- Abritrary version values at the (approx) time this was written; this don't really matter as
 -- they'll get updated within a first few seconds of initial startup.
-INSERT INTO release_versions (project, version, updated) VALUES ('oxen-io/session-desktop', 'v1.7.3', '2021-10-14Z');
-INSERT INTO release_versions (project, version, updated) VALUES ('oxen-io/session-android', '1.11.11', '2021-10-14Z');
-INSERT INTO release_versions (project, version, updated) VALUES ('oxen-io/session-ios', '1.11.17', '2021-10-14Z');
+INSERT INTO release_versions (project, version, prerelease_version, updated) VALUES ('oxen-io/session-desktop', 'v1.7.3', NULL, '2021-10-14Z');
+INSERT INTO release_versions (project, version, prerelease_version, updated) VALUES ('oxen-io/session-android', '1.11.11', NULL, '2021-10-14Z');
+INSERT INTO release_versions (project, version, prerelease_version, updated) VALUES ('oxen-io/session-ios', '1.11.17', NULL, '2021-10-14Z');
+
+CREATE TABLE release_notes (
+    project varchar(50) NOT NULL,
+    version varchar(25) NOT NULL,
+    name TEXT,
+    notes TEXT
+);
+
+CREATE INDEX release_notes_project_version ON release_notes(project, version);
+
+CREATE TABLE release_assets (
+    project varchar(50) NOT NULL,
+    version varchar(25) NOT NULL,
+    name varchar(225) NOT NULL,
+    url varchar(225) NOT NULL
+);
+
+CREATE INDEX release_assets_project_version ON release_assets(project, version);
+
+INSERT INTO release_assets (project, version, name, url) VALUES ('oxen-io/session-desktop', 'v1.7.3', 'Test name', 'github.com');
```

Comment:
I don't think this initial row is needed, as there can simply be no assets associated with a given release.

Reply:
I think this was test data I forget to remove 😓 

Have removed now